oas_docs/output/kibana.yaml

```diff
@@ -34373,6 +34373,171 @@ paths:
       x-metaTags:
         - content: Kibana
           name: product_name
+  /api/fleet/epm/packages/_bulk_rollback:
+    post:
+      description: |-
+        **Spaces method and path for this operation:**
+
+        <div><span class="operation-verb post">post</span>&nbsp;<span class="operation-path">/s/{space_id}/api/fleet/epm/packages/_bulk_rollback</span></div>
+
+        Refer to [Spaces](https://www.elastic.co/docs/deploy-manage/manage-spaces) for more information.
```

Comment:
Would this link be more useful? https://www.elastic.co/docs/api/doc/kibana/topic/topic-kibana-spaces